### PR TITLE
Loadouts

### PR DESCRIPTION
**Pull request by Lamasmaster**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Removes tool and belt tab, makes some item condensed into categories, removes scrap weapons and nerfs makeshift katana price, shuffled around and renamed some items to be reflective of actual thing
	
<hr>
</details>

## Changelog
:cl:
balance: Loadouts
/:cl:

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/lists/general.dm [hunk 1]
```diff
@@ -70,10 +83,6 @@
 	display_name = "spaceball booster pack"
 	path = /obj/item/pack/spaceball
 
-/datum/gear/trackingimplanter
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/general.dm:73]: please keep this

File: code/modules/client/preference_setup/loadout/lists/tools.dm [hunk 2]
```diff
@@ -1,88 +0,0 @@
...
-	display_name = "power cell, medium"
-	path = /obj/item/cell/medium/high
-	cost = 2
-
-/datum/gear/tools/ducttape
-	display_name = "duct tape"
-	path = /obj/item/tool/tape_roll
-	cost = 1
-
-/datum/gear/tools/flashlight
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/tools.dm:80]: Please move this somewere along with the small cell

File: code/modules/client/preference_setup/loadout/lists/utility.dm [hunk 3]
```diff
@@ -2,82 +2,61 @@
...
+	flags = GEAR_HAS_TYPE_SELECTION
+
+/datum/gear/utility/lunchbox
+	display_name = "lunch box selection"
+	path = /obj/item/storage/lunchbox
+	flags = GEAR_HAS_TYPE_SELECTION
+
+/datum/gear/utility/multipen
+	display_name = "pen selection"
+	path = /obj/item/pen
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/utility.dm:54]:
```suggestion
	path = /obj/item/pen/multi
```

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 4]
```diff
@@ -709,78 +709,3 @@
 	armor_penetration = ARMOR_PEN_DEEP
 	price_tag = 500
 	matter = list(MATERIAL_STEEL = 22, MATERIAL_WOOD = 10, MATERIAL_PLASTEEL = 4)
-
-//Shitty loadout weapons
-/obj/item/tool/cheap
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:714]: please do keep the cheap line of weapons, and keep them in loadout v_v

File: code/datums/craft/recipes/weapon.dm [hunk 5]
```diff
@@ -275,8 +275,8 @@
 	name = "makeshift katana"
 	result = /obj/item/tool/sword/katana_makeshift
 	steps = list(
-		list(/obj/item/tool/knife, 1, "time" = 15),
-		list(/obj/item/tool/knife, 1, "time" = 15),
+		list(/obj/item/stack/rods, 3, "time" = 15),
+		list(CRAFT_MATERIAL, 7, MATERIAL_STEEL),
```
Comment (ThePainkiller) [inline code/datums/craft/recipes/weapon.dm:279]: This is a **MASSIVE** nerf, this makes the makeshift katana go from 2 steel (one per knife) to **TEN** (3 rods + 10 metal)
I hope you at least reflected this change by giving the makeshift katana `matter = list(MATERIAL_STEEL = 10)`

Comment (Lamasmaster) [reply code/datums/craft/recipes/weapon.dm:279]: Puts it in line with the other makeshift weapons, not really a nerf but seems like an oversight at best

File: code/modules/clothing/accessories/holster.dm [hunk 6]
```diff
@@ -291,17 +291,3 @@ Sword holsters
 /obj/item/clothing/accessory/holster/saber/saya/occupied/Initialize()
 	holstered = new holstered_spawn
 	update_icon()
-
-/obj/item/clothing/accessory/holster/saber/machete/cheap
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/accessories/holster.dm:295]: Holter is nice qol for trash gear and makeshift stuff

**Comment by Trilbyspaceclone** [comment]
Crowbar should also stay in case of spawning in during power outages ect.
Other then that everything at a glance looks good

**Pull request merged**